program/databases/db_tests

```diff
@@ -6986,3 +6985,10 @@
 "007325","","8","/radio.php","GET","Login Page","","password","","","PHP backdoor found.","",""
 "007326","","1","/app-ads.txt","GET","DIRECT","google.com,","","","","The ads.txt file can reveal information about Google Adwords or other advertising networks in use.","",""
 "007327","RFC-5785","3","/.well-known/jwks","GET","keys","sig","","","","JWKS file found","",""
+"100001","setup_php","1","/setup.php","GET","200","","","","","The setup.php may contain sensitive informations such as users and credentials.","",""
+"100002","nginx","1","/nginx.conf","GET","200","","\s*server\s*\{(?:[^{}]|\{[^{}]*\})*\s*\}","","","Nginx web server, may reveal details about the server configuration","",""
+"100003","instructions_php","1","/instructions.php","GET","200","","","","","The instructions.php may expose informations about the configuration of the server.","",""
+"100004","NEW001","1","/php.ini","GET","200","","= (Off|On|off|on)\s*","","","The php.ini may contain important php settings.","","","","",""
```

Comment:
```suggestion
"100004","","1","/php.ini","GET","200","","= (Off|On|off|on)\s*","","","The php.ini may contain important php settings.","","","","",""
```
<img width="414" alt="Screenshot 2024-05-30 at 19 32 42" src="https://github.com/pentesttoolscom/nikto/assets/102951386/00ef3197-45f5-4e85-ad30-8b220bdd30c7">

NEW001 was test id, not reference